src/mykrobe/cortex/mccortex.py

```diff
@@ -40,55 +40,59 @@ def syscall(command):
     logger.debug(f"stderr:\n{completed_process.stderr.rstrip()}")
     return completed_process
 
-class McCortexRunner(object):
 
+class McCortexRunner(object):
     def __init__(self):
-        dir_of_this_file = os.path.dirname(os.path.abspath(__file__))
-        self.mccortex31_path = os.path.join(dir_of_this_file, "mccortex31")
+        if "MYKROBE_MCCORTEX" in os.environ:
```

Comment:
Given this mccortex envvar string is repeated a couple of times, it might be safer to add it to constants

Reply:
Good idea, changed to do that.